app/controllers/api/v1/invitations_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::InvitationsController < Api::V1::ApiController
+  def create
+  	token = SecureRandom.hex(10)
+  	WebsiteMember.create(:website_id => params[:member][:website][:id], :invitation_token => token, :invitation_sent_at => Time.now.utc )
```

Comment:
1. nu ai access la metoda `current_website`?
1. de ce creezi tokenul aici in controller?
1. de ce nu verifici daca mai exista tokenul asta?
    1. daca mai exista inca o invitatie cu acelasi token o sa dai approve foarte aiurea cand da vre-unu sa faca confirm.

---

Foloseste before_create in modelul website_member si acolo faci loop si setezi tokenul. Aici vreau sa apelezi `current_website.members.create(invitation_sent_at: Time.now.utc)`